### ISSUE TITLE
replace class in program with new program and import it and use it

### ISSUE THREAD

**Issue by MalleSrikanth** [MalleSrikanth/class-replace-with-new-program-in-python#1]
can i replace class code written in program with use of self and __init__ , with new program in it and
 import it and use it with out self and __init__

class BatchNorm(KL.BatchNormalization):
    """Extends the Keras BatchNormalization class to allow a central place
    to make changes if needed.

    Batch normalization has a negative effect on training if batches are small
    so this layer is often frozen (via setting in Config class) and functions
    as linear layer.
    """
    def call(self, inputs, training=None):
        """
        Note about training values:
            None: Train BN layers. This is the normal mode
            False: Freeze BN layers. Good when batch size is small
            True: (don't use). Set layer in training mode even when making inferences
        """
        return super(self.__class__, self).call(inputs, training=training)
